packages/suite/src/support/messages.ts

```diff
@@ -5696,6 +5700,10 @@ const messages = defineMessagesWithTypeCheck({
         defaultMessage: '{amount} selected',
         description: 'Number of list items selected',
     },
+    TR_COLLECTION_NAME_OF_TOKEN_ID: {
+        id: 'TR_COLLECTION_NAME_OF_TOKEN_ID',
+        defaultMessage: '{tokenValue}x {collectionName} of Token ID: ',
```

Comment:
Can we add also id here?

Reply:
No, because I need this to be a string to feed it to BlurUrls  that only accepts strings, and ID must be wrapped into a TrezorLink component, so here I won't use it. I think it's ok like that

Reply:
Okay, then please remove the space after Token ID: for both strings. I am scared that the translation agency would remove it anyway and it would be broken in some languages